iodata/basis.py

```diff
@@ -367,3 +367,26 @@ def get_default_conventions():
 
 
 HORTON2_CONVENTIONS, PSI4_CONVENTIONS = get_default_conventions()
+
+# fmt:off
+CCA_CONVENTIONS = {
```

Comment:
These are not yet used, which is fine, I guess? Just a few quick questions:
- Is the `fmt:off` related to black?
- Could you add a reference for these data? There may be some ambiguity. As far as I recall, CCA conventions only apply to Cartesian functions, not pure ones.

Reply:
fmt:off is for black, I mostly used black throughout but not for this section, so that it didn't take 100 lines. 

I sourced the CCA conventions from the [libint wiki](https://github.com/evaleev/libint/wiki/using-modern-CPlusPlus-API#solid-harmonic-gaussians-ordering-and-normalization)

Reply:
Thanks. I forgot about the Wiki.